container/compliance/process_results.py

```
@@ -78,37 +108,347 @@ def write_csv(packages: list[dict[str, str]], output_path: Path | None) -> None:
+    compliance container image. Using importlib.util.spec_from_file_location keeps
+    this script independent of package layout.
+    """
+    if not file_path.is_file():
+        return None
+    spec = importlib.util.spec_from_file_location(module_name, str(file_path))
+    if spec is None or spec.loader is None:
+        return None
+    module = importlib.util.module_from_spec(spec)
+    try:
+        spec.loader.exec_module(module)  # type: ignore[union-attr]
+    except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)